tests-blackbox/common/variables.ts

```diff
@@ -0,0 +1,98 @@
+import { PrimaryKeyType } from './types';
```

Comment:
Should we rely on `import { PrimaryKey } from '@directus/shared/types';` instead?

Reply:
Nope. The `PrimaryKeyType` consists of the following string literals `'integer' | 'uuid' | 'string'`.